circuits-circom/circuits/garanti/garanti_send.circom

```diff
@@ -95,6 +95,10 @@ template GarantiSendEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
     var max_payer_mobile_num_packed_bytes = count_packed(max_payer_mobile_num_len, pack_size);
     assert(max_payer_mobile_num_packed_bytes < max_body_bytes);
 
+    var max_payee_name_len = 42; // International SWIFT standard for beneficiary name is 35 characters. We add some padding for special chars.
```

Comment:
In this, each special character is counted as 1. Whereas, we would be counting it as 2. So, technically the max length should be at least 70 to account for all special characters. But average turkish names are 11 to 17 characters in length. So we should be fine.